packages/web/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.1",
   "scripts": {
     "start": "cross-env PORT=8080 react-scripts start",
+    "start:https": "cross-env HTTPS=true PORT=8081 react-scripts start",
```

Comment:
Let's maybe use port 4330, the same one as we use on dev env on the server, just for synchronization, and not to remember too many of them?
Currently on EC2:

```
web@HTTP:
dev:    8080
stage:  8081
prod:   80

web@HTTPS:
dev:    4430
stage:  4431
prod:   443
```

I'm not saying it's perfect, but least we could repeat the same locally to lower the mental burden :)

Reply:
Changed, tnx